docker-app/qfieldcloud/subscription/models.py

```diff
@@ -602,6 +603,7 @@ def active_users(self):
 
     @property
     def active_users_count(self) -> int:
+        # TODO this should be a reliable count of active users, else needs to be renamed
```

Comment:
@suricactus eventhough there are tests describing the behaviour I do not really understand this. In the `active_users()` we set a default if `current_period_until` is not set and in the `active_users_count()` we return null. E.g. I get two users returned from `active_users` and 0 from `active_users_count()`. Not sure which one to use for the admin list_display and actually not even sure whether my calculation in https://github.com/opengisch/qfieldcloud/pull/704 was correct then ... ? At least we should rename the method to what it really does, e.g. `active_users_amount_for_billing` or so?